wiki/Help_Centre/Account_Restrictions/id.md

```diff
@@ -1,15 +1,10 @@
----
-outdated: true
-outdated_since: 2b3040251f1f4876dba46e9c7e01045a7c8ccfc0
----
+# Akun Restriction
```

Comment:
```suggestion
# Restriction Akun
```
with indonesian language grammar, Restriction Akun is the correct one. i won't fix the rest lol